specification/loadtestservice/data-plane/Microsoft.LoadTestService/stable/2022-11-01/loadtestservice.json

```diff
@@ -594,7 +594,7 @@
           "Test"
         ],
         "summary": "Configure server metrics for a test",
-        "operationId": "Test_CreateOrUpdateServerMetricsConfig",
+        "operationId": "Test_CreateOrUpdateTestServerMetricsConfig",
```

Comment:
similar to AppComponents, I think, this should also be `Test_CreateOrUpdateServerMetricsConfig`